working/augmentation-libraries/feature-specification.md

```diff
@@ -304,39 +303,39 @@ It is a compile-time error if:
     original declaration occurs, according to merge order. *An augmentation
     library can both declare a new declaration and augment it in the same file.*
 
-### Augment super
+### Augmented Expression
 
-The exact result of an `augment super` expression depends on what is being
+The exact result of an `augmented` expression depends on what is being
 augmented, but it follows generally the same rules as any normal identifier:
 
-*   **Augmenting getters**: Within an augmenting getter `augment super` invokes
-    the getter and evaluates to the return value. If augmenting a field with a
+*   **Augmenting getters**: Within an augmenting getter `augmented` invokes the
+    getter and evaluates to the return value. If augmenting a field with a
     getter, this will invoke the implicit getter from the augmented field.
 
-*   **Augmenting setters**: Within an augmenting setter `augment super` must be
+*   **Augmenting setters**: Within an augmenting setter `augmented` must be
     followed by an `=` and will directly invoke the augmented setter. If
     augmenting a field with a setter, this will invoke the implicit setter from
     the augmented field.
 
-*   **Augmenting fields**: Within an augmenting field, `augment super` can only
-    be used in an initializer expression, and refers to the original field's
+*   **Augmenting fields**: Within an augmenting field, `augmented` can only be
+    used in an initializer expression, and refers to the original field's
     initializer expression, which is immediately evaluated.
 
-    It is a compile-time error to use `augment super` in an augmenting field's
+    It is a compile-time error to use `augmented` in an augmenting field's
     initializer if the member being augmented is not a field with an
     initializer.
 
-*   **Augmenting functions**: When augmenting a function, `augment super` refers
-    to the augmented function. Tear offs are allowed.
+*   **Augmenting functions**: When augmenting a function, `augmented` refers to
+    the augmented function. Tear offs are allowed.
```

Comment:
As discussed this morning, let's say that tear-offs are not allowed. The `augment` keyword must be followed by a parenthesized argument list (with optionally a preceding type argument list).

We probably also need to specify what happens if you don't pass the right number, types, or parameter names to the augmented function.

Reply:
> We probably also need to specify what happens if you don't pass the right number, types, or parameter names to the augmented function.

Is that not implicit?

Reply:
Maybe? It's probably fine.